### DISCUSSION TITLE
node_modules/node-sass: command failed.

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [nodejs/help#4652] [Q&A]
Recently I clone a project and when I tried to run it -yarn, it's throwing the error - node_modules/node-sass: command failed. I delete the node_modules folder and try to run it again but failed. Even though I downgrade my node version to 14 (https://stackoverflow.com/questions/60394291/error-node-modules-node-sass-command-failed?page=1&tab=scoredesc#tab-top), I upgrade the node-sass": "^5.0.0" to 7, even though I upgrade the whole package but failed to run it.

The repo which I try to run locally (https://github.com/bashleigh/another-portfolio)
Screen Shorts are attached, Please help me to solve it. 

![error](https://user-images.githubusercontent.com/31897843/223347706-a338867f-832f-4358-8b26-5e0a80607517.png)
![error last](https://user-images.githubusercontent.com/31897843/223347732-63e2cb4f-8363-44a9-8365-b10398db59a8.png)
![dep](https://user-images.githubusercontent.com/31897843/223347759-56e4b6d2-c024-444a-b549-e05d5533a6ff.png)
![npm](https://user-images.githubusercontent.com/31897843/223347798-1b44f4bf-a1c7-4ee1-9905-8dbb6a0c87b2.png)

**Comment by aduh95** [comment]
Sharing screenshots is a bad practice for accessibility, as your local color settings can be hard to decipher for others (e.g. folks with visual deficiencies). It's also more data to download for those who want to help you, which is not nice for those of us who have a limited connection. Always prefer copy pasting the text inside code blocks.

Did you see that message: `You need to install the latest version of Visual Studio including the "Desktop development with C++" workload. For more information consult the documentation at https: //github.com/nodejs/node-gyp#on-windows` in the stack trace? Did you try to follow that link?

**Comment by lynally** [comment]
https://stackoverflow.com/questions/60394291/error-node-modules-node-sass-command-failed
your node version is not  match to node-sass version.
